babel.config.json

```diff
@@ -9,7 +9,7 @@
       }
     ],
     "@babel/preset-typescript",
-    "@babel/preset-react"
+    ["@babel/preset-react", { "runtime": "automatic"},],
```

Comment:
Minor: It doesn't seem to be an issue for the build so maybe this is parsed as the [JSON5](https://json5.org/) variant of JSON, but the extra commas here would be a syntax error for vanilla JSON.

```suggestion
    ["@babel/preset-react", { "runtime": "automatic" }]
```

Reply:
resolved with commit 7a94ead9419f8b2d41251ef625f19676121398de